src/modules/Auth/components/Registration/Registration.css

```diff
@@ -0,0 +1,223 @@
+.primary-color{
+  background-color: #EF9B28 ;
+}
+
+.floating-form  {
+  display: flex;
+  flex-direction: column;
+  background-color: #314a59d4;
+  margin: auto;
+  max-width: 1400px;
+  padding: 15px 55px;
+  border-radius: 15px;
+}
+.logo-container{
+  text-align: center;
+}
+.img-fluid{
+  max-width: 100%;
+  height: auto;
+  width: 31%;
+}
+.auth-text-header{
+  line-height: 0.1;
+  color: #EF9B28 ;
+  position: relative;
+  font-weight: 700;
+}
+
+.auth-text-header::first-letter {
+  display: inline-block; 
+  border-bottom: 3px solid; 
+
+}
+
+
+  @media (min-width: 1200px) {
+    .form-inputs .username {
+      order: 1; /* Username input first */
```

Comment:
I don't recommend depending on the flex order as it is not practical and will have impact in keyboard navigation. We should instead depend on the DOM order and the items that we want first make them first in the DOM order.

Reply:
done